internal_cert_management/scripts/lego.sh

```
@@ -0,0 +1,64 @@
+# Validate required variables
+: "${ADMIN_EMAIL:?ADMIN_EMAIL is not set}"
+: "${SUBDOMAINS_EC256:?SUBDOMAINS_EC256 is not set}"
+: "${SUBDOMAINS_RSA2048:?SUBDOMAINS_RSA2048 is not set}"
+
+# Convert space-separated strings to arrays
+read -ra subdomains_ec256 <<< "$SUBDOMAINS_EC256"
+read -ra subdomains_rsa2048 <<< "$SUBDOMAINS_RSA2048"
+
+for type in ec256
+do
+    for subdomain in ${subdomains_ec256[@]}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)